rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -2069,7 +2070,13 @@ extern void ppu_finalize(const ppu_module& info)
 
 extern void ppu_precompile(std::vector<std::string>& dir_queue, std::vector<lv2_prx*>* loaded_prx)
 {
-	// Remove duplicates
+	// Make sure we only have one '/' at the end and remove duplicates.
+	for (std::string& dir : dir_queue)
+	{
+		while (dir.back() == '/' || dir.back() == '\\')
+			dir.pop_back();
```

Comment:
Maybe try dir.erase(dir.begin() + (dir.find_last_not_of(fs::delim) +1), dir.end());

Reply:
This doesn't make much sense since we already manually add /

Reply:
Its about not calling pop_back potentially many times.

Reply:
I can do it twice for / and \.
But what's the big difference for a couple of dirs